Code/Editor/AzAssetBrowser/AzAssetBrowserRequestHandler.cpp

```diff
@@ -372,6 +373,77 @@ void AzAssetBrowserRequestHandler::AddContextMenuActions(QWidget* caller, QMenu*
+            }
+            else
+            {
+                // Be more specific with errors so as to give the user the best chance at fixing them
+                if (!info.HasFlag(AzToolsFramework::SCF_OpenByUser))
+                {
+                    if (info.HasFlag(AzToolsFramework::SourceControlFlags::SCF_OutOfDate))
+                    {
+                        QMessageBox::information(
+                            AzToolsFramework::GetActiveWindow(), QObject::tr("Asset not deleted"),
+                            QObject::tr("Source Control Issue - You do not have latest changes from source control for file\n%1")
+                                .arg(fullFilePath.c_str()));
+                    }
+                    else if (info.IsLockedByOther())
+                    {
+                        QMessageBox::information(
+                            AzToolsFramework::GetActiveWindow(), QObject::tr("Asset not deleted"),
+                            QObject::tr("Source Control Issue - File exclusively opened by another user %1 ->\n%2")
+                                .arg(info.m_StatusUser.c_str(), fullFilePath.c_str()));
+                    }
+                    else if (
+                        info.m_status == AzToolsFramework::SourceControlStatus::SCS_ProviderIsDown ||
+                        info.m_status == AzToolsFramework::SourceControlStatus::SCS_CertificateInvalid ||
+                        info.m_status == AzToolsFramework::SourceControlStatus::SCS_ProviderError)
+                    {
+                        QMessageBox::information(
+                            AzToolsFramework::GetActiveWindow(), QObject::tr("Asset not deleted"),
+                            QObject::tr("Source Control Issue - Failed to remove file from source control, check your connection to your "
+                                    "source control service.\n%1")
+                                .arg(fullFilePath.c_str()));
+                    }
+                    else
+                    {
+                        QMessageBox::information(
+                            AzToolsFramework::GetActiveWindow(), QObject::tr("Asset not deleted"),
+                            QObject::tr("Unknown Issue with source control.\n%1").arg(fullFilePath.c_str()));
+                    }
+                }
+                else
+                {
+                    QMessageBox::information(
+                        AzToolsFramework::GetActiveWindow(), QObject::tr("Asset not deleted"),
+                        QObject::tr("Source Control Issue - File marked as 'Open By User' but still failed.\n%1").arg(fullFilePath.c_str()));
+                }
+            }
+        };
+        // Add Delete option
+        menu->addAction("Delete asset", [fullFilePath, callback]()
+        {
+            QMessageBox box;
+            box.setIcon(QMessageBox::Warning);
+            box.setWindowTitle("Delete selected asset?");
```

Comment:
This should be wrapped in a `QObject::tr` as well since it's a user-facing string

Reply:
Done.